### PR TITLE
Update and expand the CRSF and CRST Telem pages

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-crsf-telemetry.rst [hunk 1]
```diff
@@ -11,8 +11,34 @@ OpenTx will discover the native CRSF Telemetry sensors, which then can be displa
 
 .. image:: ../../../images/crossfire-telemetry-meaning.jpg
 
+These values can be displayed on OPenTX transmitters in several ways:
```
Comment (andyp1per) [inline common/source/docs/common-crsf-telemetry.rst:14]: OpenTX

File: common/source/docs/common-crsf-telemetry.rst [hunk 2]
```diff
@@ -11,8 +11,34 @@ OpenTx will discover the native CRSF Telemetry sensors, which then can be displa
...
+
+Selecting the Crossfire Configuration LUA script in the transmitter will show:
+
+.. image:: ../../../images/crsf-config-screen.png
+
+And selecting the ArduPilot vehicle shown in that list will activate the :ref:`common-paramosd` with a list of all parameters which have been setup for both OSD screens. 
+
+.. image:: ../../../images/crsf-param-editor.png
 
+.. note:: Some autopilots will not display the parameter values that have text names, as text,but rather as a number, in order to save flash space. See :ref:`common-limited-firmware` for those without CRSF TEXT capability.
```
Comment (andyp1per) [inline common/source/docs/common-crsf-telemetry.rst:44]: "as text, but" (needs the extra space)

**Review by andyp1per — APPROVED** [review]

File: common/source/docs/common-crsf-telemetry.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-crsf-telemetry.rst]: fixed

File: common/source/docs/common-crsf-telemetry.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-crsf-telemetry.rst]: fixed

**Comment by Hwurzburg** [comment]
thanks...merging

**Pull request merged**